connector/src/main/java/org/geysermc/connector/command/defaults/DumpCommand.java

```diff
@@ -141,6 +143,6 @@ public void execute(GeyserSession session, CommandSender sender, String[] args)
 
     @Override
     public List<String> getSubCommands() {
-        return Arrays.asList("offline", "full");
+        return Arrays.asList("offline", "full", "logs");
```

Comment:
imo it wouldn't hurt to also have "log" as a case, just as a catch for people who make a typo/forget